cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionUtils.java

```diff
@@ -360,7 +359,7 @@ public static void processAlterPartitionReassignmentsResult(AlterPartitionReassi
     for (Map.Entry<TopicPartition, KafkaFuture<Void>> entry: result.values().entrySet()) {
       TopicPartition tp = entry.getKey();
       try {
-        entry.getValue().get(EXECUTION_TASK_FUTURE_ERROR_VERIFICATION_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+        entry.getValue().get();
```

Comment:
With this change, an  `ExecutionException ee`, where `org.apache.kafka.common.errors.TimeoutException.class == ee.getCause().getClass()` is`true` should not be quietly handled with a `warn` as is.

Instead, in such a case, the code should throw an `IllegalStateException`, urging the user to increase the value of this new AdminClient timeout config (i.e. it should refer to static variable with the config name) if needed.